functest/deposit_transfer_test.go

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// +build functest
+
+package functest
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDepositTransferToken(t *testing.T) {
```

Comment:
What this test checks? Trying to do transfer 10 times and check user's balance after?

Reply:
It waits hold period. You can transfer only after hold period. Hold period is 6 months but I set it to 1 pulse